node/src/components/consensus/highway_core/state.rs

```diff
@@ -146,11 +146,8 @@ pub(crate) struct State<C: Context> {
     faults: HashMap<ValidatorIndex, Fault<C>>,
     /// The full panorama, corresponding to the complete protocol state.
     panorama: Panorama<C>,
-    /// Panorama used when creating new units.
-    /// In the presence of faults may lag behind `panorama`.
-    /// Units, that if cited by a new unit would make that unit violate the LNC, are not added to
-    /// `citable_panorama`.
-    citable_panorama: Panorama<C>,
+    /// The merged panoramas of all endorsed units. This is always safe to cite.
+    endorsed_panorama: Panorama<C>,
```

Comment:
Are all units really endorsed or some just don't need to b/c they don't cite any equivocator?

Reply:
I see, this has changed.

Reply:
Exactly: Those are really only the endorsed units and those cited by them.